laravel/database/factories/EventFactory.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
-use Carbon\Carbon;
 use App\Models\Event;
+use Carbon\Carbon;
+use Faker\Generator as Faker;
 
-$factory->define(Event::class, function (Faker\Generator $faker, $data)
+$factory->define(Event::class, function (Faker $faker, array $data)
 {
-    $start_date = (isset($data['start_date']) ? $data['start_date'] : Carbon::now());
+    $start_datetime = Carbon::tomorrow();
+    $end_datetime = $start_datetime->copy()->addWeeks($faker->randomDigitNotNull);
```

Comment:
This is a really long duration for a default. Most events are only 3-7 days long, not weeks.